packages/wp-now/src/start-server.ts

```diff
@@ -0,0 +1,85 @@
+import WPNow from "./wp-now"
+import { HTTPMethod } from "@php-wasm/node";
+import express from 'express';
+import fileUpload from 'express-fileupload';
+import { DEFAULT_PORT } from "./constants";
+
+
+function generateMultipartFormDataString(json, boundary) {
+  let multipartData = '';
+  const eol = '\r\n';
+
+  for (const key in json) {
+    multipartData += `--${boundary}${eol}`;
+    multipartData += `Content-Disposition: form-data; name="${key}"${eol}${eol}`;
+    multipartData += `${json[key]}${eol}`;
+  }
+
+  multipartData += `--${boundary}--${eol}`;
```

Comment:
why not add the final boundary outside of the function then? or create a kind of `multipartJoinSegments` function so we don't do this?

I guess I was just a little confused because the function name suggests to me that it's a generic function to create multiple multi-part chunks.